internal/k8stest/k8s_collector.go

```diff
@@ -20,7 +20,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-func CreateCollectorObjects(t *testing.T, client *K8sClient, testID string, manifestsDir string) []*unstructured.Unstructured {
+func CreateCollectorObjects(t *testing.T, client *K8sClient, testID string, manifestsDir string, containerRegistry string) []*unstructured.Unstructured {
```

Comment:
Instead of the new param here for the container registry, couldn't we do the same thing as in `client.go` where we try to read the env variable here? Then we would save from the calling code to pass it.

Reply:
yes good point - will adapt that